examples~/regression-tests/server/StdbModule.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="SpacetimeDB.Runtime" Version="1.1.0" />
```

Comment:
This version doesn't exist on nuget so running `dotnet build` locally doesn't work.